pages/_app.tsx

```diff
@@ -39,29 +39,28 @@ export default function App(props: AppProps) {
   return (
     <>
       <Head>
-        <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1" />
-
         <meta name="description" content={siteConfig.description} />
-        <meta name="theme-color" media="(prefers-color-scheme: light)" content="white" />
-        <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#163861" />
+        <meta name="theme-color" media="(prefers-color-scheme: light)" content="black" />
+        <meta name="theme-color" media="(prefers-color-scheme: dark)" content="black" />
 
         <link rel="shortcut icon" type="image/png" href="/favicon.png" />
-        <link rel="apple-touch-icon" sizes="192x192" href="/images/logo-square-192.png" />
-        <link rel="apple-touch-icon" sizes="512x512" href="/images/logo-square-512.png" />
+        <link rel="apple-touch-icon" sizes="192x192" href="/favicon.png" />
```

Comment:
Is it ok to use the 512x512 one here?

Reply:
For simplicity now I'd say yes. But for future iterations we could use a guide like this one https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs (tl:dr; mostly offsetting to a webmanifest file and using specific sizes)